### ISSUE TITLE
Correction for "Migrating and Upgrading Projects" storage script

### ISSUE THREAD

**Issue by tourbillonlabs** [supabase/supabase#13794]
# Improve documentation

## Link 

https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#migrate-storage-objects

## Describe the problem

Code as it is on the page errors out. Needs the correct options to specify schema.

## Describe the improvement

Script for "Migrate Storage Objects" section currently shows:

```
const oldSupabaseRestClient = createClient(OLD_PROJECT_URL, OLD_PROJECT_SERVICE_KEY, {
    schema: 'storage',
})
```

Based on docs, and that this correction allows the script to work, it should be:

```
const oldSupabaseRestClient = createClient(OLD_PROJECT_URL, OLD_PROJECT_SERVICE_KEY, {
    db: {
        schema: 'storage'
    },
})
```

**Comment by MildTomato** [comment]
@soedirgo @sweatybridge can you confirm above?

**Comment by soedirgo** [comment]
Yup can confirm the section is currently outdated - thanks @tourbillonlabs!

**Comment by GaryAustin1** [comment]
Documents were updated.
